docs/NOW-spec.md

```diff
@@ -951,7 +629,7 @@ The NOW_SESSION_MSGBOX_RSP_MSG is a message sent in response to NOW_SESSION_MSGB
             <td colspan="32">requestId</td>
         </tr>
         <tr>
-            <td colspan="32">response</td>
+            <td colspan="32">status</td>
```

Comment:
Use new NOW_STATUS instead of simple response value to report errors if message box request has failed